build.js

```diff
@@ -5,11 +5,11 @@ if (os.type() === "Linux") {
   console.log("Skipping iOS Dependencies");
 } else if (os.type() === "Darwin") {
   const downloadProcess = exec(
-    "./scripts/download-purchases-framework.sh 3.0.1"
+    "./scripts/download-purchases-framework.sh 3.1.1"
```

Comment:
gonna update this soon to 3.1.2 so that it includes the `setPushTokenString` method from 3.1.2